tests/bots/test_unvetter.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from unittest.mock import Mock
+
+import pytest
+
+from bots.unvetter import UnvetterBot
+from tests.fixtures import upgrade_staking_router_to_v2
+from transport.msg_types.unvet import UnvetMessage
+
+
+# WARNING: These accounts, and their private keys, are publicly known.
+COUNCIL_ADDRESS = '0x70997970C51812dc3A010C7d01b50e0d17dc79C8'
+COUNCIL_PK = '0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d'
+
+
+def get_unvet_message(web3) -> UnvetMessage:
+    latest = web3.eth.get_block('latest')
+    block_number = latest.number
+
+    prefix = web3.lido.deposit_security_module.get_unvet_message_prefix()
+    nonce = web3.lido.staking_router.functions.getStakingModuleNonce(1).call()
+
+    msg_hash = web3.solidity_keccak(
+        ['bytes32', 'uint256', 'uint256', 'uint256[]', 'uint256[]'],
+        [prefix, block_number, nonce, [1], [1]],
```

Comment:
why dont you sign block hash and saking module id too ?

Reply:
fixed!